toolkits/global/packages/global-ratings-survey/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@springernature/global-ratings-survey",
+  "version": "0.0.1",
+  "license": "MIT",
+  "description": "html form based component for gathering user rating data",
+  "keywords": [],
+  "author": "Springer Nature",
+  "brandContext": "^29.0.3",
+  "dependencies": {
+    "@springernature/global-forms": "^5.0.0"
```

Comment:
As a single installable thing I'm guessing because we are using the `global-forms` as a dependency here we get the `brand-context` by default, because it's a dependency of the `global-forms`?

Reply:
Yes good point. And consequently there is a possibility that at any time this component effectively has a dependency on two different versions of Brand Context at the same time 😬 (this package.json and Global Form's package.json)

Removing the dependency on Global Forms would result in the entire pictographic radios implementation being duplicated in to this package though. Which seems very unappealing.

Hmm 🤔

Reply:
The `demo/index.html` is using brand-context stuff -- so I think we can leave this as it is. 

Personally, I think it's better to have components consume other components than duplicating things because we cannot do that. 

hashtag resolved.